Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
We should also update Newstonsoft.Json in here to `13.0.1` like what is being done in `EdFi.Ods.Profiles.Sample.csproj` .

Reply:
`Autofac.Extensions.DependencyInjection` was also updated to `7.2.0` as part of the net6 changes in the API so we should update it to the same version here.

Reply:
yes updated